test/unit/connect-inject-deployment.bats

```diff
@@ -683,6 +683,16 @@ load _helpers
 #--------------------------------------------------------------------
 # namespaces
 
+@test "connectInject/Deployment: fails if namespaces are disabled and mirroringK8S is true" {
+  cd `chart_dir`
+  run helm template \
+      -s templates/connect-inject-deployment.yaml  \
```

Comment:
I know namespaces defaults to `false`, but can we set it to false explicitly in this test for clarity? Or rename the testcase to explicitly state namespace aren't enabled?

Reply:
done.

Reply:
Thanks!!